### PR TITLE
labhub.py: Add private response

### PR DESCRIPTION
**Pull request by sladyn98**
Added user mentions in labhub commands
Added a user mention in lmgtfy plugin
as well

Closes https://github.com/coala/corobo/issues/349

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

**Comment by sladyn98** [comment]
@Vamshi99 @abhishalya Can you guys kindly give this a look.Thanks

File: plugins/labhub.py [hunk 1]
```diff
@@ -409,17 +409,19 @@ def eligible(user, iss):
             if not iss.assignees:
                 if eligible(user, iss):
                     iss.assign(user)
-                    yield ('Congratulations! You\'ve been assigned to the '
-                           'issue. :tada:')
+                    yield ('@{}, congratulations! You\'ve been assigned to the '
+                           'issue. :tada:'.format(user))
                 else:
-                    yield 'You are not eligible to be assigned to this issue.'
+                    yield ('@{}, You are not eligible to be assigned to this'
```
Comment (abhishalya) [inline plugins/labhub.py]: Missing space after `this` causes the test to fail.

```suggestion
                    yield ('@{}, You are not eligible to be assigned to this '
```

Comment (sladyn98) [reply plugins/labhub.py]: Yeah @abhishalya, small mistakes :P

File: plugins/labhub.py [hunk 2]
```diff
@@ -333,7 +333,7 @@ def assign_cmd(self, msg, match):
         try:
             assert org == self.GH_ORG_NAME or org == self.GL_ORG_NAME
         except AssertionError:
-            yield 'Repository not owned by our org.'
+            yield '@{}, repository not owned by our org.'.format(user)
```
Comment (abhishalya) [inline plugins/labhub.py]: R -> r, hence should be changed in the test too

**Comment by sladyn98** [comment]
@abhishalya I fixed the tests

**Comment by abhishalya** [comment]
>@abhishalya I fixed the tests

Are you sure?

**Comment by sladyn98** [comment]
Yeah pretty sure

**Comment by abhishalya** [comment]
Can you test the `@username`, changing the case isn't helping much. It would be pretty simple to do since I think we already have examples like those.

**Comment by sladyn98** [comment]
@abhishalya I guess the test is self sufficient.

**Pull request closed without merging**